src/OneDrive.Sdk.Authentication.Desktop/Business/AuthenticationContextWrapper.cs

```diff
@@ -54,6 +54,19 @@ public ITokenCache TokenCache
             }
         }
 
+
+        /// <summary>
+        /// Authenticates the daemon app silently"/>.
+        /// </summary>
+        /// <param name="resource">The resource to authenticate against.</param>
+        /// <param name="clientCredential">The client credential of the application.</param>
+        /// <returns>The <see cref="IAuthenticationResult"/>.</returns>
+        public async Task<IAuthenticationResult> AcquireDaemonTokenSilentAsync(string resource, ClientCredential clientCredential)
```

Comment:
That's what I asked you why AcquireTokenSilentAsync doesn't work. It throws exception.
Daemon process don't require end user ID anyway.